Susi/Controllers/ChatViewController/SpeechRecognizerDelegate.swift

```diff
@@ -67,9 +67,7 @@ extension ChatViewController: SFSpeechRecognizerDelegate, AVSpeechSynthesizerDel
 
         recognitionRequest = SFSpeechAudioBufferRecognitionRequest()
 
-        guard let inputNode = audioEngine.inputNode else {
-            fatalError("Audio engine has no input node")
-        }
+        let inputNode = audioEngine.inputNode
```

Comment:
Any specific reason for this change?

Reply:
`inputNode` is defined as let `audioEngine = AVAudioEngine()` which is not optional. when we use guard let it is giving obvious error: 
> Initializer for conditional binding must have Optional type, not 'AVAudioInputNode'